### ISSUE TITLE
How to bind to multiple refs easily?

### ISSUE THREAD

**Issue by codeofsumit** [posva/vuexfire#106]
In your ToDo example you bind to all ToDos available in the collection.
As per firebase best practices, if I have ToDos specific to users, I save the ID of a users ToDo also in the user collection.
So now I have 5 ToDo-IDs in a user collection, and a ToDo collection of 1000 ToDos.

How to I bind the 5 ToDo-IDs of a user from the ToDo collection?
Basically this:
https://stackoverflow.com/questions/38192711/how-to-retrieve-multiple-keys-in-firebase

**Comment by posva** [comment]
I'd actually do the other way around: specify who is the owner of the todo **on** the todo object and then being able of specifying a Query with 

```js
ref.child('todos').orderByChild('owner').equalTo(userId)
```

If you still want to go the other way, you will have to bind every of the five todos, and probably build a getter that returns them as an array

**Comment by codeofsumit** [comment]
I got it both ways. I got the owner on the todos, and the todos on the user as you don't want to search 10 million entries for a user that has 2 ;-)

Anyway, I'm trying to bind all of the todos (seems to be the right way as recommended by firebase and you) but how can I pass a sub property of the state to `bindFirebaseRef` ?

```
const state = {
  todo: {},
};

// firebaseAction etc inside a loop, an id is passed:
bindFirebaseRef('todo.'+id, ref, {
  readyCallback: (val) => {
    console.log(val);
  },
});
```

This doesn't seem to work to bind a ref to a specific todo, to `state.todo.id`.
Can I do this somehow?
